<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RA2Mem</class>
 <widget class="QWidget" name="RA2Mem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>713</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RA2_Data analysis</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>经济消耗</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>经济剩余</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>导入数据</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWebEngineView" name="webview1" native="true">
         <property name="minimumSize">
          <size>
           <width>1366</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>2560</width>
           <height>1440</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>当前状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>程序启动中</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
